<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="fullLayout">
     <item>
      <layout class="QHBoxLayout" name="DeviceInfoLayout">
       <property name="spacing">
        <number>50</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="devListLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="devListLabel">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Device List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="devTable">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>4</height>
            </size>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column/>
           <column/>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="reSearchLayout">
           <item>
            <widget class="QPushButton" name="reloadBtn">
             <property name="text">
              <string>Reload</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="devInfoLayout">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="devInfoLabel">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Device Information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="devInfo"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="btnLayout">
           <property name="spacing">
            <number>7</number>
           </property>
           <item>
            <widget class="QPushButton" name="policyBtn">
             <property name="text">
              <string>Policy</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deleteBtn">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
